tests/e2e/sample-applications/mysql-persistent/mysql-persistent-csi.yaml

```diff
@@ -173,7 +177,7 @@ items:
         spec:
           containers:
           - name: todolist
-            image: quay.io/konveyor/todolist-mariadb-go:v2_4
+            image: quay.io/migtools/oadp-ci-todolist-mariadb-go:latest
```

Comment:
why this change?

Reply:
To keep all our images in one place which we have access to, also built those apps using newest go version.